src/commcare_cloud/ansible/roles/deploy_hq/tasks/run_webpack.yml

```diff
@@ -1,10 +1,20 @@
 - name: Check for webpack configuration
+  # We can eventually remove this when all of HQ is on Webpack
   stat:
-    path: "{{ code_source }}/webpack.config.js"
-  register: config_file 
+    path: "{{ code_source }}/webpack"
+  register: webpack_config_directory
 
-- name: Run webpack
+- name: Generate Webpack Settings
+  # generates webpack/settings.json so that webpack has the same path
+  # to STATIC_ROOT as django
   command:
-    cmd: yarn run build:webpack
+    cmd: ./manage.py generate_webpack_settings
     chdir: '{{ code_source }}'
-  when: config_file.stat.exists
+  when: webpack_config_directory.stat.exists
+
+- name: Build Webpack Bundles
+  # Builds all the webpack bundles
+  command:
+    cmd: yarn build
+    chdir: '{{ code_source }}'
+  when: webpack_config_directory.stat.exists
```

Comment:
The duplication of the `webpack_config_directory.stat.exists` conditional doesn't seem ideal. Is this something we can get around with [blocks](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_blocks.html)? Another link would be https://stackoverflow.com/a/34541646